examples/quantizing_moe_fp8/tinymixtral_4_248_moe.py

```diff
@@ -0,0 +1,29 @@
+from llmcompressor.modifiers.quantization.gptq import GPTQModifier
+from llmcompressor.transformers import oneshot
+
+# model to quantize
+model = "Isotonic/TinyMixtral-4x248M-MoE"
```

Comment:
Does this model give valid results at all? I'm thinking it might make sense to use a larger model like mistralai/Mixtral-8x7B-Instruct-v0.1 (We would then also want to enable cpu offloading and run GPTQ sequentially)